### DISCUSSION TITLE
Llama-3-Chinese-8B-Instruct-v2 GGUF 对于 system message 为什么是忽略的？

### DISCUSSION THREAD

**Discussion by zungmou** [ymcui/Chinese-LLaMA-Alpaca-3#41] [Q&A]
无论是 Ollama: /set system 还是 LM Studio 都一概忽视 system message设定，请问是版本问题还是？

**Comment by ymcui** [comment]
我试了一下没有问题：ollama 0.1.37 + ggml-model-q8_0.gguf （inst-v2版）

<img width="833" alt="image" src="https://github.com/ymcui/Chinese-LLaMA-Alpaca-3/assets/16095339/52fa699b-e081-45ef-8d54-b216c1019365">
